packages/editor-ui/src/hooks/posthog.ts

```diff
@@ -0,0 +1,426 @@
+// version 0.1.0
+
```

Comment:
reminder to add a comment to the top of these files that these are deprecated

Reply:
Will add to GitLab hooks file.